### DISCUSSION TITLE
RTE actions are not working after the first time

### DISCUSSION THREAD

**Discussion by DhanaDSP1120** [GrapesJS/grapesjs#4549] [Q&A]
Hi..!, the RTE actions are not working after the first time , when we try to apply the action on previous applied element its not working

It occurs on version 0.18.3 to till latest , its working fine on below versions

Reproduce links

Working version : 
https://jsfiddle.net/76gnxh4d/35/

Not working version :
https://jsfiddle.net/76gnxh4d/36/

Demo :
![grapesjs version issue](https://user-images.githubusercontent.com/54067384/188276505-70b8ead9-f0e4-4d82-ae31-09ae478ef468.gif)

**Comment by artf** [comment]
I'll try to fix that for the next release

**Reply by DhanaDSP1120** [reply]
same version issue for [this also ](https://github.com/artf/grapesjs/issues/4558)

**Comment by DhanaDSP1120** [comment]
@artf  Not Fixed fully only font color is fixed , issue still remains in font background color
